### PR TITLE
Completed css-layout-classes exercise

### PR DESCRIPTION
**Pull request by wang-morris**
Screen Shot 2022-05-23 at 8 48 17 PM

### CODE REVIEW

File: css-layout-classes/index.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-full">
+        <h1>Choose your character!</h1>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-third">
```
Comment (scrumptiousdev) [inline css-layout-classes/index.html]: `.column-third` is acting as flex container like `.row` already does. Do we need two flex containers nested in another? Same for line 29 as well.

**Review by scrumptiousdev — CHANGES_REQUESTED** [review]
Small change is needed

**Comment by wang-morris** [comment]
Screen Shot 2022-05-23 at 9 02 47 PM

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**